### ISSUE TITLE
Run rake task as a job

### ISSUE THREAD

**Issue by plashchynski** [plashchynski/crono#10]

**Comment by michaelachrisco** [comment]
```Ruby
# cronotab.rb
require 'rake'
AppName::Application.load_tasks

class Test
  def perform
    Rake::Task['crono:hello'].invoke
  end
end

Crono.perform(Test).every 5.seconds
```
With the rake task of:
```Ruby
#lib/tasks/test.rake
namespace :crono do
  desc 'Update all tables'
  task :hello => :environment do
    puts "hello"
  end
end
```

**Comment by plashchynski** [comment]
@michaelachrisco Thank you, that's good variant. Closed by #15

**Comment by karthikdc88** [comment]
Hi, i have put this code in my project. I am planning on running a email generation code using rake. But in the above example how do i run the crono gem. How do i make it run so every 5 sec it will give me a output of hello. Does the hello appear on the terminal?

Please help me out i am stuck at this for my project.
